command/docker_logger_plugin.go

```diff
@@ -25,12 +25,19 @@ func (e *DockerLoggerPluginCommand) Synopsis() string {
 }
 
 func (e *DockerLoggerPluginCommand) Run(args []string) int {
+	logger := log.New(&log.LoggerOptions{
+		Level:      log.Trace,
+		JSONFormat: true,
```

Comment:
Will this cause interweaving of JSON and non-JSON formatted logs in the output of `json_logs` is false?

Reply:
No it will not (https://github.com/hashicorp/go-plugin/blob/master/client.go#L936). Go-plugin essentially parses the json logs and then logs it back out to the client's logger. So the format will be whatever the client has